gcloud/storage/batch.py

```diff
@@ -170,10 +170,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 def _unpack_batch_response(response, content):
     """Convert response, content -> [(status, reason, payload)]."""
     parser = Parser()
-    faux_message = ('Content-Type: %s\nMIME-Version: 1.0\n\n%s' %
-                    (response['content-type'], content))
-
-    message = parser.parsestr(faux_message)
+    if not isinstance(content, six.binary_type):
```

Comment:
Yeah that would work too. I was just trying to fix the `%s` bug with `bytes` in Py3 and didn't focus too much on the fix.

On the flip side, using `str` scares me. I'd rather be explicit. (Isn't this a Py2/Py3 best practice anway? e.g. Brett Cannon's tool complains about not using `b''` and `u''`.)